internal/templates.go

```diff
@@ -942,8 +942,10 @@ spec:
                   - kube-dns
               topologyKey: kubernetes.io/hostname
       serviceAccountName: coredns
+      priorityClassName: system-cluster-critical
       tolerations:
         - key: node-role.kubernetes.io/master
```

Comment:
If we drop the key here, this will allow it to run on any node. Do we want that? I am not sure. As an example, we have ingress nodes that are tainted with `NoSchedule`. If we drop the key, then coredns could run on ingress nodes in this example.

Reply:
I believe we want to run master/worker node, but not on other types if possible